docs/admin/cluster-large.md

```diff
@@ -75,6 +75,10 @@ However, this is done only for clusters having more than 50 nodes.
 
 ### Addon Resources
 
+** TODO: Update this to reflect
+[https://github.com/kubernetes/kubernetes/pull/16185/](https://github.com/kubernetes/kubernetes/pull/16185/)
+and any other changes since it was written. **
+
 To prevent memory leaks or other resource issues in [cluster addons](../../cluster/addons/) from consuming all the resources available on a node, Kubernetes sets resource limits on addon containers to limit the CPU and Memory resources they can consume (See PR [#10653](http://pr.k8s.io/10653/files) and [#10778](http://pr.k8s.io/10778/files)).
```

Comment:
We did really hardcap CPU since we didn't config the kernel with CONFIG_FAIR_GROUP_SCHED. If we do nothing for addons after #21086, the statement is truely reflect the situation this time. If I decide to make addons burstable at the end, we should change the statement here. But anyway, we are not worse than today. :-P